pkgs/servers/monitoring/nagios-plugins/openbsd_snmp3_check/default.nix

```
@@ -0,0 +1,48 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  openbsd_snmp3_check,
+  python3,
+  stdenv,
+  testers,
+}:
+stdenv.mkDerivation rec {
+  pname = "openbsd_snmp3_check";
+  version = "0.55";
+
+  src = fetchFromGitHub {
+    owner = "alexander-naumov";
+    repo = "openbsd_snmp3_check";
+    rev = "v${version}";
+    hash = "sha256-qDYANMvQU72f9wz8os7S1PfBH08AAqhtWLHVuSmkub4=";
+  };
+
+  buildInputs = [
+    python3
+  ];
```

I believe the supported way to package a simple Python script is to use `pyproject = false`:

```suggestion
python3Packages.buildPythonApplication rec {
  pname = "openbsd_snmp3_check";
  version = "0.55";
  pyproject = false;

  src = fetchFromGitHub {
    owner = "alexander-naumov";
    repo = "openbsd_snmp3_check";
    rev = "v${version}";
    hash = "sha256-qDYANMvQU72f9wz8os7S1PfBH08AAqhtWLHVuSmkub4=";
  };
  
```

---

That seems better for this Python utility, thanks!